game/tests.py

```
@@ -1192,4 +1192,85 @@ def test_promotion_notation_invalid_piece_defaults_to_queen(self):
+        )
+
+        self.user2 = User.objects.create_user(
+            username='player2',
+            password='testpass123'
+        )
+
+    def test_create_private_room(self):
+
+        self.client.login(
+            username='player1',
+            password='testpass123'
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "password"

(S106)